pipelines/dax/docker/build_imgserv.groovy

```diff
@@ -13,28 +13,55 @@ node('jenkins-master') {
 }
 
 notify.wrap {
+  def retries = 3
+  def repo = 'webserv/imgserv'
+  def tag = 'dax_latest'
+
+  def run = {
+    dir('imgserv/lsst-dm-ci') {
+        stage('build') {
+            retry(retries) {
+                build(repo)
+            }
+        } // build
+
+        stage('test') {
+            retry(retries) {
+                docker.image(repo:tag).inside {
```

Comment:
I assume there is a missing string to be interpolated into.

Reply:
is '$repo:$tag' the correct syntax for the missing string?

Reply:
See latest. I'm now using "${repo}:${tag}" for string interpolation.

Reply:
Looks OK now.